pallets/capacity/src/lib.rs

```diff
@@ -285,7 +285,7 @@ pub mod pallet {
 	/// Reward Pool history
 	#[pallet::storage]
 	#[pallet::getter(fn get_reward_pool_for_era)]
-	pub type StakingRewardPool<T: Config> =
+	pub type ProviderBoostRewardPool<T: Config> =
```

Comment:
Maybe the plural is better, ProviderBoostRewardPools, because it stores multiple RewardPoolInfo records.